webapp/components/experiments/Experiments.js

```diff
@@ -154,80 +160,97 @@ const Experiments = (props) => {
     const date = new Date(timestamp);
     return date?.toLocaleDateString();
   };
+
+  const [dialogOpen, setDialogOpen] = useState(false)
+
+  const [list, setList] = useState(true);
+
+  const createExperimentScreen = () => {
+    setDialogOpen(false)
+    setList(false)
+  }
   return (
-    <GridLayout className={classes.root}>
-      <div>
-        <Box>
-          <TableContainer component={Paper}>
-            <Table aria-label="simple table">
-              <TableBody>
-                {experiments.map((experiment) => (
-                  <TableRow key={experiment?.name}>
-                    <TableCell component="th" scope="row">
-                      <Button>
-                        <Typography variant="h6" className="experimentHead">
-                          {experiment?.name}
+    <>
+    { list ?
```

Comment:
Ternary operators increase readability for short conditional checks `active ? doThat() : doThis()`, ideally it should fit into one line, but avoid using it across 100 lines, there is too much detail in this check

rather do: `list ? <Experiments/> : <EditExperiment/>`

Also keep in mind that we have more than two options, the detail view of an Experiment is another component